cluster-provision/images.sh

```diff
@@ -3,6 +3,8 @@
 set -e
 
 declare -A IMAGES
+IMAGES[fedora31]=""
```

Comment:
We don't need it ?

Reply:
I suspect this is a placeholder where you plan to insert the image sha.

Reply:
Exactly :)
This  is a placeholder fot the VM container image sha.

Reply:
job running, let's wait for it to finish so we have the final hash.